### PR TITLE
Update no.json

### PR DESCRIPTION
**Pull request by 0pax**
**Pull Request Description**

Added missing translations for the phone in Norwegian Bokmål. Let me know if there's any mistakes @itschip 
**Pull Request Checklist**:
* [x] Have you followed the guidelines in our Contributing document and Code of Conduct? Yes
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/project-error/npwd/pulls) for the same update/change? Yes
* [x] Have you built and tested NPWD in-game after the relevant change? Yes

### CODE REVIEW

File: phone/src/locale/no.json [hunk 1]
```diff
@@ -1,220 +1,283 @@
 {
   "translation": {
+    "MISC": {
+      "TOP_LEVEL_ERR_TITLE": "Kritisk Feil",
+      "TOP_LEVEL_ERR_ACTION": "Reload",
```
Comment (itschip) [inline phone/src/locale/no.json:5]: Restart?

File: phone/src/locale/no.json [hunk 2]
```diff
@@ -1,220 +1,283 @@
...
       "OPTIONS": {
+        "ICONSET": "Ikoner",
+        "THEME": "Tema",
         "LANGUAGE": "Språk",
-        "RESET_SETTINGS": "Reset Settings",
-        "RESET_SETTINGS_DESC": "Dette vil slette all lagret data",
+        "FRAME": "Deksel",
+        "ZOOM": "Zoom",
+        "WALLPAPER": "Bakgrunn",
+        "RESET_SETTINGS": "Reset Innstillinger",
```
Comment (itschip) [inline phone/src/locale/no.json:41]: Maybe "Fabrikkinnstill"?

File: phone/src/locale/no.json [hunk 3]
```diff
@@ -1,220 +1,283 @@
...
+    },
+    "DIALER": {
+      "MESSAGES": {
+        "CURRENT_CALL_TITLE": "Telefonsamtale",
+        "INCOMING_CALL_TITLE": "Innkommende telefonsamtale",
+        "CURRENT_CALL_WITH": "I samtale med {{ transmitter }}",
+        "TRANSMITTER_IS_CALLING": "{{ transmitter }} ringer deg!"
       },
+      "INPUT_PLACEHOLDER": "Skriv et nummer",
+      "NAVBAR_HISTORY": "History",
```
Comment (itschip) [inline phone/src/locale/no.json:93]: "Logg" / "Historikk"

File: phone/src/locale/no.json [hunk 4]
```diff
@@ -1,220 +1,283 @@
...
+        "SAVE_LIKES_FAILED": "Klarte ikke lagre likes",
+        "NEW_LIKE_FOUND": "Ny match funnet!"
+      },
+      "PROFILE_CONFIGURATION": "Appen er feilkonfigurert - kan ikke opprette profil. Vennligst kontakt en administrator.",
+      "EDIT_PROFILE_IMAGE": "Bildeurl",
+      "EDIT_PROFILE_NAME": "Profilnavn",
+      "EDIT_PROFILE_BIO": "Bio",
+      "EDIT_PROFILE_LOCATION": "Lokasjon",
+      "EDIT_PROFILE_JOB": "Jobb",
+      "EDIT_PROFILE_TAGS": "Tagger",
```
Comment (itschip) [inline phone/src/locale/no.json:157]: Would probably be "Knagger" lol, but if you want to keep it as "Tags", sure.

File: phone/src/locale/no.json (hunk 2 above)
Comment (0pax) [reply phone/src/locale/no.json:41]: Agreed, will change.

File: phone/src/locale/no.json (hunk 1 above)
Comment (0pax) [reply phone/src/locale/no.json:5]: Agreed

File: phone/src/locale/no.json (hunk 4 above)
Comment (0pax) [reply phone/src/locale/no.json:157]: "Tagger" sounds better imo, but yeah the direct translation would be "Knagger" as you said. What do you think?

Comment (itschip) [reply phone/src/locale/no.json:157]: What about just "Emneknagg", too keep it more centralized. Then whomever that want to, can change it.

Comment (0pax) [reply phone/src/locale/no.json:157]: Yeah I agree, will do.

**Comment by itschip** [comment]
Please go over the Pull Request Checklist as well :)

**Comment by 0pax** [comment]
@itschip everything is good, just forgot to check it when submitting the PR (thats why it says Yes after)

**Review by itschip — APPROVED** [review]

**Pull request merged**